vmdb/app/models/metric/processing.rb

```diff
@@ -2,6 +2,8 @@ module Metric::Processing
   DERIVED_COLS = [
     :derived_cpu_available,
     :derived_cpu_reserved,
+    :derived_logicalcpus_used,
+    :derived_logicalcpus_available,
```

Comment:
Can you explain what used and available mean here?  I understand available becaiuse it's a count of logical CPUs, but used doesn't make sense..